tests/test_sqlalchemy_data_layer.py

```diff
@@ -410,6 +421,29 @@ class PersonList(ResourceList):
     yield PersonList
 
 
+@pytest.fixture(scope="module")
+def person_list_custom_qs_manager(session, person_model, dummy_decorator, person_schema, before_create_object):
+    class QS(QSManager):
+        def _simple_filters(self, dict_):
+            return [{"name": key, "op": "in" if isinstance(value, list) else "eq", "val": value}
+                    for (key, value) in dict_.items()]
+
+    class PersonList(ResourceList):
+        schema = person_schema
+        data_layer = {
+            "model": person_model,
+            "session": session,
+            "methods": {"before_create_object": before_create_object},
```

Comment:
`methods` is not used and `before_create_object` won't be overridden, you can remove this line

Reply:
i removed it